CMakeLists.txt

```diff
@@ -123,7 +123,7 @@ endif ( )
 
 if ( SUITESPARSE_USE_SYSTEM_BTF )
     list ( REMOVE_ITEM SUITESPARSE_ENABLE_PROJECTS "btf" )
-    find_package ( BTF REQUIRED )
+    find_package ( BTF 2.3.0 REQUIRED )
```

Comment:
Does KLU really need the exact version 2.3.0 of BTF?

The intention of the `SUITESPARSE_USE_SYSTEM_*` flags was to allow users to link to an installed version of the respective library. That might also include support for partial updates.

If that doesn't work because the user installed a library with an incompatible API, that would be a user error.
I don't know if the build system should be so restrictive (to prevent any potential user error?).

Reply:
Side-note: It took me a while to notice this change. It is completely unrelated to the remainder of this commit. And there is no mention of this change in the commit message...

Reply:
It should be 2.3.0 and later.  That's what the find_package parameter implies, if my understanding is correct. Using an old BYF will get the old build system and it will likely break, so I thought it would be safest to require at least the latest packages.